ProjectSettings/ProjectSettings.asset

```diff
@@ -668,6 +670,8 @@ PlayerSettings:
   webGLThreadsSupport: 0
   scriptingDefineSymbols:
     1: 
+    4: ARFOUNDATION_PRESENT
```

Comment:
Looks like two settings files were included as well. Please remove these, as we don't want things like this scripting define checked-in.